src/matrixrtc/MembershipManager.ts

```diff
@@ -306,7 +325,7 @@ export class MembershipManager
      */
     public constructor(
         private joinConfig: (SessionConfig & MembershipConfig) | undefined,
-        private room: Pick<Room, "getLiveTimeline" | "roomId" | "getVersion">,
+        protected room: Pick<Room, "roomId" | "getVersion">,
         private client: Pick<
             MatrixClient,
             | "getUserId"
```

Comment:
This could use the proposed: `MembershipManagerClient` (see StickyEventMembershipManager)